extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/boot/QuarkusPersistenceUnitDefinition.java

```diff
@@ -195,6 +205,14 @@ public Properties getPuProperties() {
         public void setPuProperties(Properties puProperties) {
             this.puProperties = puProperties;
         }
+
+        public boolean isEnversIsPresent() {
```

Comment:
Minor nitpicking but this method name looks very weird. The boolean should be called `enversPresent` and the method `isEnversPresent`.

Reply:
:+1:  fixed and rebased.